CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ See the nixpkgs manual for more details on [standard meta-attributes](https://ni
 
 In addition to writing properly formatted commit messages, it's important to include relevant information so other developers can later understand *why* a change was made. While this information usually can be found by digging code, mailing list/Discourse archives, pull request discussions or upstream changes, it may require a lot of work.
 
-For package version upgrades and such a one-line commit message is usually sufficient.
+For package version upgrades a simple commit message indicating the old and new version with a reference to the release notes or changelog are sufficient.
```

Comment:
It is not quite clear that the headline is a must and the changelog is a should.

Reply:
I should probably change that to must be given, if one exists. Thanks for the pointer.

Reply:
The `meta.changelog` could be referenced for such things.

Reply:
Yeah, sure. Now only tools like `nix-update` and `nixpkgs-update` need to catch up in that regard.